data-products/_docs/cleansing-definitions.md

```diff
@@ -87,12 +78,22 @@ Our [example data product](./_example/) contains a few more examples.
 
 ### Data with no cleansing applied
 
-If your data has not undergone any cleansing, we suggest you explicitly inform users of this by adding `type: none` for each of your tables. Otherwise we will assume there has been undocumented cleansing. An exception is made for tables [defined as reference data](./_example/02-data-dictionary.yml#L67) - here we assume no cleansing has been applied if there is no cleansing specified.
+If your data has not undergone any cleansing, we suggest you explicitly inform users of this by adding `type: none` for each of your tables to make this clear to users. Otherwise we will assume there has been undocumented cleansing. An exception is made for tables [defined as reference data](./_example/02-data-dictionary.yml#L67) - here we assume no cleansing has been applied if there is no cleansing specified.
```

Comment:
What if you remove the reference to 'table' here?
Maybe the text could read: "by adding `type: none` to make this clear to users"